fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/BulkExportJobExecutionResponse.java

```diff
@@ -632,4 +634,12 @@ public static String generate(BulkExportJobExecutionResponse obj, boolean withSe
 
     }
 
+    public String getExitStatus() {
```

Comment:
a choice of style... 

I like to put getters/setters together in the code, and subclasses at the end of a model/bean class

Reply:
Ha, I thought they were inserted at the end of other getter/setter. let me change

Reply:
done